### PR TITLE
Machine Cleanup

### PR DESCRIPTION
**Pull request by pspoerri**

### CODE REVIEW

File: submit.lema.slurm [hunk 1]
```diff
@@ -1,17 +0,0 @@
-#!/bin/bash
```
Comment (lxavier) [inline submit.lema.slurm]: RIP lema

**Comment by pspoerri** [comment]
Should I also remove Santis and Dora? Currently we don't really support them.

**Comment by pspoerri** [comment]
I also updated the `machineEnvironment.sh` file.

**Comment by lxavier** [comment]
Dora is gone, keep Santis we maz need it again at some point

**Pull request closed without merging**